# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tqca/hotfix-installpath.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- tqca-trinity-14.1.4/CMakeLists.txt.vanilla	2025-05-14 20:38:55.103786325 +0200
+++ tqca-trinity-14.1.4/CMakeLists.txt	2025-05-14 20:39:00.837054841 +0200
@@ -39,7 +39,7 @@
 ##### setup install paths
 
 if( CMAKE_INSTALL_LIBDIR )
-  tde_setup_install_path( LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" )
+  tde_setup_install_path( LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}" )
 endif( )
 
 include( TDESetupPaths )
